### DISCUSSION TITLE
Bootstrap Email Templates

### DISCUSSION THREAD

**Discussion by jeremybdk** [bootstrap-email/bootstrap-email#109] [General]
Hello,
I just wanted to know if you had a timeframe in mind for the Email Templates or already have an early access? Would love to participate (and pay of course for nice templates).

Thanks

**Comment by stuyam** [comment]
Hi Jermey! Thanks for reaching out and the support 😄 It has certainly been a long journey this year. However templates are well underway. A lot of work needed to be done to Bootstrap Email to support a more robust set of templates. Most of that work as been done and Bootstrap Email 1.0 is in it's alpha phase which is good.

Now the focus shifts back to getting the templates to where they need to be. Unfortunately I don't know exactly when that would be, however something like "early access" would be very interesting and something I might consider soon.

I will try to keep you in the loop for those coming in the hopefully near future!
